### ISSUE TITLE
[Beta3] Able to craft copper chests with copper ingots.

### ISSUE THREAD

**Issue by CrypticDivide** [Darkosto/SevTech-Ages#720]
When playing on Darkosto's sub server (running Beta3 of the pack), we (LadyEloquent, Xris_Hawkins, GameDevBrian, and I) found that you are able to craft copper chests using coppers ingots instead of copper plates. Interestingly enough, we also found that this does not work for the wood to copper chest upgrade, even though they have very similar recipes.

I have uploaded a short video to YouTube showing that crafting the copper chest with ingots does work, but it does not show that trying to use copper ingots instead of copper plates does not work for the wood to copper chest upgrade. Please note that once this issue is resolved, the YouTube video will be taken down. https://youtu.be/mTxUO0Em-Pc

Thanks! ~CrypticDivide, LadyEloquent, Xris_Hawkins, and GameDevBrian

**Comment by CrypticDivide** [comment]
I have done some testing pertaining to this issue in Beta4 of the pack.

The first thing that I did was to create a new test server without FastWorkbench, using a client that did have FastWorkbench to join it. The first thing that I did when I joined the test server was to run the following commands...

`/gamestage add @p zero`
`/gamestage add @p zero_mob`
`/gamestage add @p one`
`/gamestage add @p one_mob`

After I ran those commands, I gave myself the required materials, a vanilla crafting bench, 8 copper ingots from Immersive Engineering, and a vanilla chest. I then tried to craft a copper chest, and was **_successful_**.

The other thing that I tried was to join the same server, except with a client that did not have FastWorkbench. I gave myself another set of the same materials (described above), and attempted to craft a copper chest, being **_successful_** yet again.

_On a side note, when I joined the test server using a client that did have FastWorkbench, I noticed that the vanilla crafting bench had a missing texture._
